### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by Red-Sage** [cvbubbles/nonlocal-3dtracking#3]
Do you intend for this project to have a license so others can use it? I'm sorry if I missed it but I don't see one.

**Comment by cvbubbles** [comment]
Yes, we have not added the license file, but it should be in MIT license, so you can use the code for free.
